packages/components/src/styles/flag/flag.stories.ts

```diff
@@ -0,0 +1,160 @@
+import '../../solid-components';
+import { html } from 'lit';
+
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+
+const { argTypes, parameters } = storybookDefaults('sd-flag');
+const { overrideArgs } = storybookHelpers('sd-flag');
+const { generateTemplate } = storybookTemplate('sd-flag');
+
+/**
+ * A small, non-interactive label the represents a category.
+ *
+ * <b>Variants</b><br>
+ * <li>--neutral-200 is the default variant</li>
+ * <li>--neutral-500</li>
+ * <li>--neutral-300</li>
+ * <li>--white</li>
+ */
+
+export default {
+  title: 'Styles/sd-flag',
+  component: 'sd-flag',
+  parameters: {
+    ...parameters,
+    backgrounds: {
+      default: 'neutral-200'
+    },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/vQ57OHDm26QE1gtZKtfb6L/Flag?type=design&node-id=804-717&mode=design&t=bdhcOj9ub57hEPZl-0'
+    }
+  },
+  args: overrideArgs([{ type: 'slot', name: 'default', value: 'Lorem Ipsum' }]),
+  argTypes
+};
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({
+      options: { templateContent: '<span class="%CLASSES%">%SLOT%</span>' },
+      args
+    });
+  }
+};
+
+export const Variants = {
+  parameters: { controls: { exclude: ['default', 'sd-flag--...'] } },
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        y: [
+          {
+            type: 'attribute',
+            name: 'sd-flag',
+            values: ['sd-flag--neutral-500', 'sd-flag--neutral-300', 'sd-flag--neutral-200', 'sd-flag--white']
+          }
+        ]
+      },
+      args
+    });
+  }
+};
+
+export const FlagSamples = {
```

Comment:
pls hide the controls in this story as they dont have any effect. or make them change the shown components... ;-)

Reply:
BTW: the same appears in the sd-chip story. Pls also have a look into this story.

Reply:
TBH I agree that it looks correct in the code. No idea why it does not work in the story...

@solid-design-system/development any idea whats wrong here?